test-framework/common/src/main/java/io/quarkus/test/common/DefaultDockerContainerLauncher.java

```diff
@@ -78,12 +84,14 @@ public void start() throws IOException {
         }
 
         List<String> args = new ArrayList<>();
-        args.add("docker"); // TODO: determine this dynamically?
+        args.add(DOCKER_BINARY); // TODO: determine this dynamically?
         args.add("run");
         if (!argLine.isEmpty()) {
             args.addAll(argLine);
         }
-        args.add("--rm");
```

Comment:
Shouldn't we still keep `--rm` ? If there is an issue in the JVM process the cleanup command may not be run.

Reply:
Yeah, it won't hurt. I'll add it

Reply:
Added it back